### PR TITLE
EVAKA-TECH: functional feature flags

### PR DESCRIPTION
**Pull request by Joosakur**
#### Summary
- rename mobileDailyNotes flag to employeeMobile and promote from experimental
- remove static pilot unit list and move under feature flag functions

### CODE REVIEW

File: frontend/src/employee-frontend/components/Header.tsx [hunk 1]
```diff
@@ -208,7 +208,7 @@ const Header = React.memo(function Header({ location }: RouteComponentProps) {
               </RequireRole>
 
               {featureFlags.messaging &&
-                (featureFlags.experimental?.mobileDailyNotes ||
+                (featureFlags.experimental?.employeeMobile ||
                   hasPilotAccess) && (
```
Comment (jaahon) [inline frontend/src/employee-frontend/components/Header.tsx:212]: You probably forgot the change this conditional, just `hasPilotAccess` should be enough.

Comment (Joosakur) [reply frontend/src/employee-frontend/components/Header.tsx]: Yeah, actually tried a slightly different approach there now.

File: frontend/src/employee-frontend/components/Header.tsx [hunk 2]
```diff
@@ -207,23 +207,21 @@ const Header = React.memo(function Header({ location }: RouteComponentProps) {
...
-                      data-qa="messages-nav"
-                    >
-                      {i18n.header.messages}
-                      {unreadCount > 0 && (
-                        <UnreadCount>{unreadCount}</UnreadCount>
-                      )}
-                    </NavbarLink>
-                  </RequireRole>
-                )}
+              {accounts.isSuccess && accounts.value.length > 0 && (
```
Comment (jaahon) [inline frontend/src/employee-frontend/components/Header.tsx:210]: I could be wrong, but doesn't every employee and group have an account?

Comment (Joosakur) [reply frontend/src/employee-frontend/components/Header.tsx:210]: Yeah need to check with @mapehe, I'm not sure how this should work. The account list excludes group accounts outside pilot units, but if every emploee has a personal account then this won't work.

**Pull request closed without merging**